internal/gojs/run/gojs.go

```diff
@@ -11,33 +11,26 @@ import (
 	"github.com/tetratelabs/wazero/sys"
 )
 
-func RunAndReturnState(
-	ctx context.Context,
-	r wazero.Runtime,
-	compiled wazero.CompiledModule,
-	moduleConfig wazero.ModuleConfig,
-	config *config.Config,
-) (*gojs.State, error) {
```

Comment:
first use is getting rid of this return, which I implemented with a notifier instead